app/components/home.jsx

```diff
@@ -29,6 +29,7 @@ export function Home () {
         <div className="sub">
           <h2>A tool for tracking your Living Dex progress! We currently support:</h2>
           <ul>
+            <li><h2>Pokémon Brilliant Diamond &amp; Shining Pearl <Link alt="Sample Brilliant Diamond Regional Living Dex" className="link" to="/u/ashketchum10/brilliant-diamond-regional-living-dex">Regional Dex</Link> and <Link alt="Sample Brilliant Diamond National Living Dex" className="link" to="/u/ashketchum10/brilliant-diamond-national-living-dex">National Dex</Link></h2></li>
```

Comment:
<img width="853" alt="Screen Shot 2022-01-27 at 8 20 15 AM" src="https://user-images.githubusercontent.com/5498072/151464076-16bf0bff-c17e-4cb3-889b-e8fffe68d3a2.png">

does this look ok?

Reply:
yup, looks great! ✅